glean-core/ios/Glean/Glean.swift

```diff
@@ -263,6 +263,64 @@ public class Glean {
         }
     }
 
+    /// When applications are launched using the custom URL scheme, this helper function will process
+    /// the URL and parse the debug commands
```

Comment:
Document what a proper URL will look like and what query parameters it parses out.

Reply:
Sure!